windows/CodePushInstallMode.cs

```diff
@@ -0,0 +1,6 @@
+﻿enum CodePushInstallMode
```

Comment:
We should wrap this in our namespace, and I think we should strip the "CodePush" prefix, since it isn't necessary with the namespace.